src/main/kotlin/com/fwdekker/randomness/ErrorReporter.kt

```diff
@@ -121,7 +137,7 @@ class ErrorReporter : ErrorReportSubmitter() {
      * @return the version number of Randomness, or `null` if it could not be determined
      */
     private fun getPluginVersion() =
-        if (pluginDescriptor is IdeaPluginDescriptor) (pluginDescriptor as IdeaPluginDescriptor).version
+        if (pluginDescriptor is IdeaPluginDescriptorImpl) (pluginDescriptor as IdeaPluginDescriptorImpl).version
```

Comment:
Prevents a warning about `#getVersion` not being available pre-2020. This is because this method has been removed from `IdeaPluginDescriptor` and has been added in `PluginDescriptor`. By downcasting the `pluginDescriptor` even further, the method once again becomes available. Once support for 2019 is dropped, this cast can be removed entirely.